public/editor/scripts/project/project.js

```diff
@@ -246,12 +247,33 @@ define(function(require) {
 
             // Look for an HTML file to open, ideally index.html
             var indexPos = 0;
+            var foundIndexHTML = false;
             found.forEach(function(path, idx) {
               if(Path.basename(path) === "index.html") {
                 indexPos = idx;
+                foundIndexHTML = true;
               }
             });
 
+            // Create a default index.html file
+            if (!foundIndexHTML) {
+              var location = "/default-files/html.txt";
+              $.get(location).then(function(data) {
+                var file = getRoot() + "/index.html";
+                _fs.writeFile(file, data, function(err) {
+                  if (err) {
+                    return console.error("Cannot write file to project: " + err);
+                  }
```

Comment:
I understand what you mean now and I just made the change to the code. Can you take a look?